examples/spark/README.md

```diff
@@ -37,23 +37,25 @@ Following this example, you will create a functional [Apache
 Spark](http://spark.apache.org/) cluster using Kubernetes and
 [Docker](http://docker.io).
 
-You will setup a Spark master service and a set of
-Spark workers using Spark's [standalone mode](http://spark.apache.org/docs/latest/spark-standalone.html).
+You will setup a Spark master service and a set of Spark workers using Spark's [standalone mode](http://spark.apache.org/docs/latest/spark-standalone.html).
 
 For the impatient expert, jump straight to the [tl;dr](#tldr)
 section.
 
 ### Sources
 
-The Docker images are heavily based on https://github.com/mattf/docker-spark
+The Docker images are heavily based on https://github.com/mattf/docker-spark.
+And are curated in https://github.com/kubernetes/application-images/tree/master/spark
 
 ## Step Zero: Prerequisites
 
-This example assumes you have a Kubernetes cluster installed and
-running, and that you have installed the ```kubectl``` command line
-tool somewhere in your path. Please see the [getting
-started](../../docs/getting-started-guides/) for installation
-instructions for your platform.
+This example assumes
+
+- You have a Kubernetes cluster installed and running.
+- That you have installed the ```kubectl``` command line tool somewhere in your path.
+- That a spark-master service which spins up will be automatically discoverable by your kube DNS impl, as 'spark-master'
```

Comment:
This bullet doesn't very clearly state "The SkyDNS add-on is running" to me, whereas 90% of the complaints I've seen, that's what they need to know. I was actually planning on putting up a PR that just checked in the Dockerfile (i.e. tried to resolve `spark-master`, complained that DNS wasn't running.)